labconnect/main/auth_routes.py

```diff
@@ -220,15 +220,18 @@ def promoteUser(email: str) -> Response:
             user_id=manager.id
         ).first()
     
+    if management_permissions.admin == True:
```

Comment:
+1 on the Ruff (E712) item, python allows and recommends just doing `if <boolean>` instead of `== True`